unit_test/entities/test_esdl_object.py

```diff
@@ -79,27 +78,25 @@ def test_get_connected_assets(self):
         self.assertEqual(connected_assets1, test_list1)
         self.assertEqual(connected_assets2, test_list2)
 
-    def test_get_asset_parameters(self):
+    def test_get_property(self):
+        """Test get_property method."""
         # Arrange
-        pipes = self.esdl_object.get_all_assets_of_type("pipe")
+        asset = self.esdl_object.get_all_assets_of_type("pipe")[0]
         # Act
-        pipe_parameters = pipes[0].get_asset_parameters()
+        length, property_available = asset.get_property("length", 0.0)
         # Assert
-        self.assertTrue(pipe_parameters["length"] == pipes[0].esdl_asset.length)
+        assert length == asset.esdl_asset.length
+        assert property_available is True
```

Comment:
Dit ziet er raar uit. je kunt toch gewoon de is True weglaten?

Reply:
Resolved